manifests/plugin/dynflow/params.pp

```diff
@@ -2,4 +2,5 @@
 class foreman_proxy::plugin::dynflow::params {
   $enabled           = true
   $listen_on         = 'https'
+  $database_path     = '/var/spool/foreman-proxy/dynflow/dynflow.sqlite'
```

Comment:
You might want to create this path in both RPM and deb packages first unless it can handle the directory not existing.

It's also worth noting that if this is intended to be persisted indefinitely then it should be /var/lib, and /var/spool should just be used for pending work.